[2024-05-19T00:07:15.826+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T00:07:16.767+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Ingest_Data_from_Redfin scheduled__2024-05-18T12:03:00+00:00 [queued]>
[2024-05-19T00:07:17.154+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Ingest_Data_from_Redfin scheduled__2024-05-18T12:03:00+00:00 [queued]>
[2024-05-19T00:07:17.169+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 6
[2024-05-19T00:07:17.649+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Ingest_Data_from_Redfin> on 2024-05-18 12:03:00+00:00
[2024-05-19T00:07:18.054+0000] {standard_task_runner.py:63} INFO - Started process 214 to run task
[2024-05-19T00:07:18.282+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Real_Estate_Webscraping_ETL', 'Ingest_Data_from_Redfin', 'scheduled__2024-05-18T12:03:00+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/webscrapping.py', '--cfg-path', '/tmp/tmp4v3g8vit']
[2024-05-19T00:07:18.447+0000] {standard_task_runner.py:91} INFO - Job 432: Subtask Ingest_Data_from_Redfin
[2024-05-19T00:07:26.491+0000] {task_command.py:426} INFO - Running <TaskInstance: Real_Estate_Webscraping_ETL.Ingest_Data_from_Redfin scheduled__2024-05-18T12:03:00+00:00 [running]> on host eddd84e36bad
[2024-05-19T00:07:34.061+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='svc' AIRFLOW_CTX_DAG_ID='Real_Estate_Webscraping_ETL' AIRFLOW_CTX_TASK_ID='Ingest_Data_from_Redfin' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T12:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-18T12:03:00+00:00'
[2024-05-19T00:07:34.210+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T00:07:34.948+0000] {conn.py:362} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.96.5', 9092)]>: connecting to kafka:9092 [('192.168.96.5', 9092) IPv4]
[2024-05-19T00:07:34.986+0000] {conn.py:957} INFO - Probing node bootstrap-0 broker version
[2024-05-19T00:07:35.026+0000] {conn.py:393} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.96.5', 9092)]>: Connection complete.
[2024-05-19T00:07:35.358+0000] {conn.py:1019} INFO - Broker version identified as 2.6.0
[2024-05-19T00:07:35.363+0000] {conn.py:1020} INFO - Set configuration api_version=(2, 6, 0) to skip auto check_version requests on startup
[2024-05-19T00:07:45.497+0000] {logging_mixin.py:188} INFO - Connecting to Scraping Browser
[2024-05-19T00:07:48.086+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T00:07:48.218+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/includes/main.py", line 208, in ingestion_start
    asyncio.run(main())
  File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/includes/main.py", line 205, in main
    await run(playwright, producer)
  File "/opt/airflow/dags/includes/main.py", line 95, in run
    browser = await pw.chromium.connect_over_cdp(SBR_WS_CDP)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/playwright/async_api/_generated.py", line 14239, in connect_over_cdp
    await self._impl_obj.connect_over_cdp(
  File "/home/airflow/.local/lib/python3.11/site-packages/playwright/_impl/_browser_type.py", line 172, in connect_over_cdp
    response = await self._channel.send_return_as_dict("connectOverCDP", params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/playwright/_impl/_connection.py", line 64, in send_return_as_dict
    return await self._connection.wrap_api_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/playwright/_impl/_connection.py", line 513, in wrap_api_call
    raise rewrite_error(error, f"{parsed_st['apiName']}: {error}") from None
playwright._impl._errors.Error: BrowserType.connect_over_cdp: WebSocket error: wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ 403 Auth Failed (customer_suspended)
Account is suspended
Call log:
<ws connecting> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/
  - <ws unexpected response> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ 403 Auth Failed (customer_suspended)
Account is suspended
  - <ws error> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ error WebSocket was closed before the connection was established
  - <ws connect error> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ WebSocket was closed before the connection was established
  - <ws disconnected> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ code=1006 reason=

[2024-05-19T00:07:48.696+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=Real_Estate_Webscraping_ETL, task_id=Ingest_Data_from_Redfin, run_id=scheduled__2024-05-18T12:03:00+00:00, execution_date=20240518T120300, start_date=20240519T000716, end_date=20240519T000748
[2024-05-19T00:07:49.403+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 432 for task Ingest_Data_from_Redfin (BrowserType.connect_over_cdp: WebSocket error: wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ 403 Auth Failed (customer_suspended)
Account is suspended
Call log:
<ws connecting> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/
  - <ws unexpected response> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ 403 Auth Failed (customer_suspended)
Account is suspended
  - <ws error> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ error WebSocket was closed before the connection was established
  - <ws connect error> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ WebSocket was closed before the connection was established
  - <ws disconnected> wss://brd-customer-hl_a024015e-zone-scraping_browser1:7kafv86s8ewk@brd.superproxy.io:9222/ code=1006 reason=
; 214)
[2024-05-19T00:07:50.672+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T00:07:50.833+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
